rpcserver.go

```diff
@@ -6048,6 +6049,21 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 
 			blindingRestrictions.NodeOmissionSet.Add(vertex)
 		}
+
+		blindingRestrictions.IncomingChainedChannels = append(
+			blindingRestrictions.IncomingChainedChannels,
+			blindCfg.IncomingChannelList...,
+		)
+
+		// When selecting the blinded incoming channel list parameter
+		// the num of hops is implictitly set.
+		numHops := blindingRestrictions.NumHops
+		numChainnedChannels :=
+			uint8(len(blindingRestrictions.IncomingChainedChannels))
```

Comment:
ahh you set it here, but you need to set `MinDistanceFromIntroNode` because we want to let the user be able to also contruct bigger routes than the chainedChannels., But you need to make sure NumHops is at least greater or equal to `MinDistanceFromIntroNode`

Reply:
Absolutely ... thanks!